### PR TITLE
fix: don't download 100gb onto local python machine in load test

### PR DESCRIPTION
**Pull request by milkshakeiii**
Thank you for opening a Pull Request! Before submitting your PR, there are a few things you can do to make sure it goes smoothly:
- [ ] Make sure to open an issue as a [bug/issue](https://github.com/googleapis/python-bigquery-dataframes/issues/new/choose) before writing your code!  That way we can discuss the change, evaluate designs, and agree on the general idea
- [ ] Ensure the tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

Fixes #<issue_number_goes_here> 🦕

### CODE REVIEW

File: tests/system/load/test_large_tables.py [hunk 1]
```diff
@@ -74,13 +74,9 @@ def test_index_repr_large_table():
...
 
 
-# FAILED
-# tests/system/load/test_large_tables.py::test_to_pandas_batches_large_table
-# google.api_core.exceptions.Forbidden: 403 Response too large to return.
-# Consider specifying a destination table in your job...
-@pytest.mark.xfail
 def test_to_pandas_batches_large_table():
-    df = bpd.read_gbq("load_testing.scalars_100gb")
+    df = bpd.read_gbq("load_testing.scalars_1gb")
```
Comment (shobsi) [inline tests/system/load/test_large_tables.py:78]: can we take a middle value, say 20gb to still represent a large table. I suspect even pandas may be able to handle 1gb.

Comment (milkshakeiii) [reply tests/system/load/test_large_tables.py:78]: I think a middle value would probably work, but I don't think we're really testing hardly anything here other than the download speed of the local machine? Even if it was an error in read_gbq, we wouldn't catch that. The middle layer is the BigQuery python library and then it's Bigframes. I don't quite see what load-related errors we're catching? 1 GB is enough to test the round trip... more seems like just testing downloading, what do you think?

Comment (shobsi) [reply tests/system/load/test_large_tables.py:78]: my understanding is that it's really the load - more the better. I agree it depends on the size of the VM running the test, but we should test as large as we can.

Comment (milkshakeiii) [reply tests/system/load/test_large_tables.py:78]: Hm, alright, well the available middle ground is 10GB, so I'll switch to that. We also have other tests that do 1TB and such in this file, so we should be all set with this change, I think.

**Review by shobsi — APPROVED** [review]

**Review by tswast — APPROVED** [review]

**Pull request merged**